project/Dependencies.scala

```diff
@@ -129,7 +130,7 @@ object Dependencies {
   private val akkaHttpSprayJson = "com.typesafe.akka" %% "akka-http-spray-json" % Versions.AkkaHttp
   private val akkaParsing       = "com.typesafe.akka" %% "akka-parsing"         % Versions.AkkaHttp
 
-  private val sprayJson = "io.spray" %% "spray-json" % "1.3.5"
+  private val sprayJson = "io.spray" %% "spray-json" % "1.3.5" force()
```

Comment:
Should we use `dependencyOverrides` here since `force` is [not recommended](https://www.scala-sbt.org/1.x/docs/Library-Management.html#Forcing+a+revision+%28Not+recommended%29)?

Reply:
I've never seen that before.

Let's see if the force is actually needed.